tests/network/vmi_istio.go

```diff
@@ -248,21 +250,21 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 			})
 		})
 		Describe("Inbound traffic", func() {
-			checkVMIReachability := func(vmi *v1.VirtualMachineInstance, targetPort int) error {
+			checkVMIReachability := func(vmi *v1.VirtualMachineInstance, targetPort int) (*batchv1.Job, error) {
```

Comment:
I don't understand why you now return the `Job`. AFAICT, all the usages just discard it.

Reply:
Ops, yeah, I was messing with afterAll instead of tests.CleanBeforeTests(), this is a remnant of that, I'll remove it